File: httpcsharp/httpcsharp/models/Author.cs
namespace httpcsharp.models
{
    public class Author
    {
        public string Id { get; }
        public string Name { get; set; }

        public Author(string id, string name)
        {
            Id = id;
            Name = name;
        }

        public bool IsValid()
        {
            if (string.IsNullOrEmpty(Name))
            {
                return false;
            }

            return true;
        }
    }
}

File: httpjava/src/main/java/org/stuartaroth/httpjava/models/Author.java
package org.stuartaroth.httpjava.models;

public class Author {
    private String id;
    private String name;

    public Author(String id, String name) {
        this.id = id;
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public boolean isValid() {
        if (name == null || name.equals("")) {
            return false;
        }

        return true;
    }
}
